cmake/modules/Findkeyutils.cmake

```diff
@@ -13,7 +13,7 @@ INCLUDE(CheckCXXSymbolExists)
 
 # include dir
 
-find_path(KEYUTILS_INCLUDE_DIR keyutils.h NO_DEFAULT_PATH PATHS
+find_path(KEYUTILS_INCLUDE_DIR keyutils.h PATHS
```

Comment:
@bachp to get this PR backported, you might need to create a tracker ticket and update the backport field with "kraken". then connect your commit with that ticket using the notation of "Fixes: http://tracker.ceph.com/issues/12345" in your commit message.